typescript/packages/jumble/integration/utils.ts

```diff
@@ -39,3 +39,68 @@ export const login = async (page: Page) => {
   button = await page.$("button");
   await button!.click();
 };
+
+export const addCharm = async () => {
+  const space = `ci-${Date.now()}-${
+    Math.random().toString(36).substring(2, 15)
+  }`;
+  const process = new Deno.Command("deno", {
+    args: [
+      "task",
+      "start",
+      "--space",
+      space,
+      "--recipeFile",
+      "recipes/simpleValue.tsx",
+      "--cause",
+      "ci",
+      "--quit",
+      "true",
+    ],
+    env: {
+      "TOOLSHED_API_URL": "http://localhost:8000/",
```

Comment:
Could be a passed in arg near `FRONTEND_URL`, not sure if this would need configured though